src/app/components/landing/landing.component.html

```diff
@@ -1 +1,12 @@
-<p>ask the user if they want äta här or ta med</p>
+<p>Welcome to ThisFoodApp, the easiest and fastest way to order your favorite food. With ThisFoodApp, you
```

Comment:
Let’s interpolate app name also, like you did for the restaurant name :)

Reply:
This is completed along with "Lets Begin"